### ISSUE TITLE
Support variations

### ISSUE THREAD

**Issue by brawer** [khaledhosny/ots#113]
Sanitizing `fvar` and `avar` should be quite easy; `gvar` and `CFF2` might be trickier.

**Comment by taylorb-monotype** [comment]
Also add support for GPOS variations in device tables. The current process fails to validate variable font GPOS tables. 
The test should look at the delta_format first and check that it's an allowable value (1, 2, 3, or 0x8000 are valid). If it's 1, 2, or 3, proceed with the start_size > end_size test. If it's format 0x8000, it should treat start_size and end_size as a DeltaSetIndex value, where start_size is the outer index and end_size is the inner index. Ideally, it should also check that the referenced DeltaSetIndex (outer, inner) exists in the ItemVariationStore (in the GDEF table) but that's going to start to get more complex because you have to read the GDEF IVS and keep a reference to it while testing the GPOS.

In summary, the key things are: 1) format 0x8000 is now a valid format, and 2) under that format, the "start_size" (outer index) can be larger than "end_size" (inner index).

**Comment by davelab6** [comment]
@taylorb-monotype thanks bob! that's great to know :)

**Comment by khaledhosny** [comment]
Should be fixed in master, testing is appreciated.
